projects/FirstProjects/FirstProject/src/com/valentin/Main.java

```diff
@@ -0,0 +1,131 @@
+package com.valentin;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    static Scanner scanner = new Scanner(System.in);
+    static ReverseArray reverseArray=new ReverseArray();
+    static ArrClass arrClass=new ArrClass();
+    static QuickSort quickSort=new QuickSort();
+
+    /**
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        int[] commonArr={};
+
+        while (true){
```

Comment:
Same and for this task. I know that university wants such things but in practice it's a rare case user input to be taken from console.